recipe/build.sh

```diff
@@ -13,7 +13,9 @@ fi
 
 
 ./configure --prefix=$PREFIX \
-            --disable-dependency-tracking
+            --disable-dependency-tracking \
+            --enable-mpi-cxx \
+            --enable-mpi-fortran
```

Comment:
I added the enable fortran flag to make sure it is explicit (but enabled is the default if a fortran compiler is present, which it needs to be here)